src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,44 @@
 package mate.jdbc;
 
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
+    private static final String YOUR_PACKAGE = "mate.jdbc";
+    private static final Injector injector = Injector.getInstance(YOUR_PACKAGE);
+
     public static void main(String[] args) {
+        ManufacturerDao manufacturerDao
+                = (ManufacturerDao) injector.getInstance(ManufacturerDao.class);
+        Manufacturer manufacturer = new Manufacturer();
+        manufacturer.setName("Mitsubishi");
+        manufacturer.setCountry("Japan");
+        Manufacturer resultManufacturer = manufacturerDao.create(manufacturer);
+        System.out.println("Create: ");
+        System.out.println(resultManufacturer);
+
+        System.out.println("Get by id (3): ");
+        Optional<Manufacturer> optionalManufacturer = manufacturerDao.get(3L);
```

Comment:
so more understandable
```suggestion
        Optional<Manufacturer> optionalManufacturer = manufacturerDao.get(resultManufacturer.getId());
```